src/panels/lovelace/hui-view.ts

```diff
@@ -311,7 +311,7 @@ export class HUIView extends hassLocalizeLitMixin(LitElement) {
   ): void {
     const newCard = this._createCardElement(config);
     element.parentElement!.replaceChild(newCard, element);
```

Comment:
@balloob 
Wouldn't this be better?
`element.replaceWith(newCard);`

Does babel polyfill this for the not supported browsers?

Reply:
So there are two things at play with Babel. One is transpiling, where it will take new syntax like arrow functions and rewrites them to an older JavaScript (ES5) equivalent. For example `=>` becomes `function() { }.bind(this)`.

This won't work with new methods added to objects in the language. For those, you need a polyfill. We load a bunch of those in https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/entrypoints/compatibility.js but `replaceWith` is not one of them. Also, our compatibility layer is only loaded for older build, not for newer. 

Safari doesn't implement the method ([docs](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/replaceWith#Browser_compatibility)) so it would mean no longer being able to ship modern version to Safari.